packages/debug/src/browser/debug-session.tsx

```diff
@@ -73,8 +73,12 @@ export class DebugSession implements CompositeTreeElement {
         protected readonly breakpoints: BreakpointManager,
         protected readonly labelProvider: LabelProvider,
         protected readonly messages: MessageClient,
-        protected readonly fileService: FileService) {
+        protected readonly fileService: FileService,
+        protected readonly debugContributionProvider: ContributionProvider<DebugContribution>
```

Comment:
Breaking change? Should it be reflected in the `changelog`?
The same would be true for `packages/plugin-ext/src/main/browser/debug/plugin-debug-session-factory.ts` even if it is not likely that someone would extend it.